server/controllers/property.js

```diff
@@ -77,12 +77,12 @@ const propertyController = {
   },
   async changeStatus(req, res) {
     const { id } = req.params;
-    const property = await Property.getPropertybyId(parseInt(id));
+    let property = await Property.getPropertybyId(parseInt(id));
 
 
     if (property) {
       if (checkOwner(req, property)) {
-        await Property.changePropertyStatus(property.propertyid);
+        property = await Property.update('status', property.propertyid, 'sold');
         userResponse.setSuccess(200, 'property advert updated successfully', property);
         return userResponse.send(res);
```

Comment:
this return statement seems unnecessary because immediately after the if statement there's another return statement returning almost the same thing